src/Pdf/CakePdf.php

```diff
@@ -738,6 +752,40 @@ public function template($template = false, $layout = null)
     }
 
     /**
+     * Template path
+     *
+     * @param mixed $templatePath The path of the template to use
+     * @return mixed
+     */
+    public function templatePath($templatePath = false)
```

Comment:
Yeah sorry, that's what I meant, change this PR to be like others for consistency, and make a PR afterwards that cleans it up. The one can me merged now, and the other can go pending for next major release.